src/realm/sync/noinst/pending_reset_store.cpp

```diff
@@ -67,29 +67,25 @@ constexpr static std::string_view s_reset_error_code_col_name("reset_error_code"
 constexpr static std::string_view s_reset_error_msg_col_name("reset_error_msg");
 constexpr int64_t s_pending_reset_version = 2;
 
-void PendingResetStore::clear_pending_reset(const TransactionRef& wr_tr)
+bool PendingResetStore::clear_pending_reset(Group& group)
 {
-    // Write transaction required
-    REALM_ASSERT(wr_tr->get_transact_stage() == DB::TransactStage::transact_Writing);
-    auto reset_store = PendingResetStore::load_or_create_schema(wr_tr);
```

Comment:
Loading the schema here was kinda slow and it wasn't actually being used for anything.